src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -617,9 +626,9 @@ protected int askQuantity(Medical med) {
 		int qty = 0;
 		do {
 			String quantity = JOptionPane.showInputDialog(this,
-							message.toString(),
-							title.toString(),
-							JOptionPane.QUESTION_MESSAGE);
+				message.toString(),
```

Comment:
`message` is defined at line 618 as:
<pre>
StringBuilder message = new StringBuilder(med.toString());
</pre>

Then why not change this line this and remove line 618
```suggestion
				med.toString(),
```

Reply:
I guess it was there since long time preparing for a longer message... Let me remove that.